concepts/eventing/performance evaluation/nats-jetstream/results_prod_HA.md

```diff
@@ -0,0 +1,332 @@
+# High Availibiltiy (HA) Evaluation of JetStream (Kyma: Production Profile)
+
+## Table of Contents
+- Purpose
+- Test Setup
+- NATS Pods Anti-Affinity
+- Test Scenario 1: K8s Cluster Node deleted during test where stream leader NATS pod was NOT deployed
```

Comment:
```suggestion
- Test Scenario 1: Kubernetes cluster Node deleted during test where stream leader NATS Pod was NOT deployed
```
https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology#capitalized-terms